mp/src/game/shared/baseentity_shared.cpp

```diff
@@ -168,7 +168,7 @@ const Vector &CBaseEntity::WorldSpaceCenter( ) const
 }
 
 #if !defined( CLIENT_DLL )
-#define CHANGE_FLAGS(flags,newFlags) { unsigned int old = flags; flags = (newFlags); gEntList.ReportEntityFlagsChanged( this, old, flags ); }
+#define CHANGE_FLAGS(flags,newFlags) { flags = (newFlags); }
 #else
 #define CHANGE_FLAGS(flags,newFlags) (flags = (newFlags))
 #endif
```

Comment:
This macro doesn't do anything anymore so you can remove it and just make whatever calls it do the `flags = newflags` logic.